GliaWidgets/Theme/ThemeFont.swift

```diff
@@ -51,8 +55,69 @@ public struct ThemeFont {
         self.header3 = header3 ?? Font.medium(18)
         self.bodyText = bodyText ?? Font.regular(16)
         self.subtitle = subtitle ?? Font.regular(14)
-        self.mediumSubtitle = mediumSubtitle ?? Font.medium(14)
+        self.mediumSubtitle1 = mediumSubtitle1 ?? Font.medium(16)
+        self.mediumSubtitle2 = mediumSubtitle2 ?? Font.medium(14)
         self.caption = caption ?? Font.regular(12)
         self.buttonLabel = buttonLabel ?? Font.regular(16)
     }
+
+    init() {
+        let fontScaling = FontScaling.theme
+        self.header1 = fontScaling.uiFont(with: .title1) // header1 ?? Font.bold(24) // .title1
+        self.header2 = fontScaling.uiFont(with: .title2) // header2 ?? Font.regular(20) // .title2
+        self.header3 = fontScaling.uiFont(with: .title3) // header3 ?? Font.medium(18) // .title3
+        self.bodyText = fontScaling.uiFont(with: .body) // bodyText ?? Font.regular(16) // .body
+        self.subtitle = fontScaling.uiFont(with: .footnote) // subtitle ?? Font.regular(14) // .footnote
+        self.mediumSubtitle1 = fontScaling.uiFont(with: .subheadline, fontSize: 16) // medium16 ?? Font.medium(16)
+        self.mediumSubtitle2 = fontScaling.uiFont(with: .subheadline)// mediumSubtitle ?? Font.medium(14) // .subheadline
+        self.caption = fontScaling.uiFont(with: .caption1) // caption ?? Font.regular(12) // .caption1
+        self.buttonLabel = fontScaling.uiFont(with: .body)// buttonLabel ?? Font.regular(16) // .bodykmj
+    }
+}
+
+extension FontScaling {
+    static let theme = Self(
+        descriptions: .init(
+            uniqueKeysWithValues: Style
+                .allCases
+                .map { style in (style, style.themeFontDescription()) }
+        ),
+        environment: .live
+    )
+}
+
+extension FontScaling.Description {
+    init(name: FontProvider.FontName, size: Double) {
+        self.init(name: name.rawValue, size: size)
+    }
+}
+
+extension FontScaling.Style {
+    func themeFontDescription() -> FontScaling.Description {
+        #warning("TODO: remove hints-comments")
```

Comment:
Let's remove this warning? It seem it is not relevant.

Reply:
3a549712bb2f0b808caa3a37e9f44349a5442f73